public/components/trace_analytics/home.tsx

```diff
@@ -295,12 +325,21 @@ export const Home = (props: HomeProps) => {
 
   const traceColumnAction = () => {
     const tracesPath = '#/traces';
+    const dataSourceId = dataSourceMDSId[0]?.id || ''; // Default to empty string if undefined
```

Comment:
nit: we can keep the same comment format.

Reply:
And also for these well-known design, I think we can remove these comments

Reply:
Removed, thanks for feedback!